frontend/app/routes/_gcweb-app.data-unavailable.tsx

```diff
@@ -29,8 +29,31 @@ export default function DataUnavailable() {
   const { SCCH_BASE_URI } = getClientEnv();
   return (
     <>
-      <p>{t('service-unavailable')}</p>
-      <InlineLink to={t('gcweb:header.menu-dashboard.href', { baseUri: SCCH_BASE_URI })}>{t('gcweb:header.menu-dashboard.text')}</InlineLink>
+      <p className="mb-6">
+        {t('service-eligiblle')} (
+        <InlineLink id="do-you-qualify-link" to={t('do-you-qualify.href')}>
+          {t('do-you-qualify.text')}
+        </InlineLink>
+        )
```

Comment:
I feel like this block of text should be rendered like this:
"You are currently not a participant in the Canadian Dental Care Plan, to find out if you are eligible please visit [Do you qualify - Canada.ca](https://www.canada.ca/en/services/benefits/dental/dental-care-plan/qualify.html))."

Currently the link is duplicated:
"You are currently not a participant in the Canadian Dental Care Plan, to find out if you are eligible please visit Do you qualify on Canada.ca ([Do you qualify - Canada.ca](https://www.canada.ca/en/services/benefits/dental/dental-care-plan/qualify.html))"

Same comment for the "How to apply - Canada.ca" and "Contact us" links below.

Reply:
fixed